ivy/functional/ivy/sorting.py

```diff
@@ -202,7 +202,7 @@ def sort(
     --------
     With :class:`ivy.Array` input:
 
-    >>> x = ivy.array([7, 8, 6])
```

Comment:
Hey @GOW7527,
I don't know if it's an oversight,
but the order [7, 8, 6] is already in the unsorted order,

henceforth, this change is not necessary.
Thanks anyway.